shopinvader_data_stored/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* Add tests
```

Comment:
and cleanup shopinvader_search_engine when the connector_search_engine refactor will happen